### ISSUE TITLE
Jenkins build failed due to a missing file

### ISSUE THREAD

**Issue by raghavendra1221** [eirslett/frontend-maven-plugin#969]
[DEBUG] copy ReactUI/src/main/resources/web/node_modules/cross-spawn/node_modules/.bin/semver to ReactUI/target/classes/web/node_modules/cross-spawn/node_modules/.bin/semver

I am having an issue where the plugin copies files from Installation directory to target directory and it fails because it is missing a file as shown as above. I am not sure why that file is not getting generated or missed. Is there a way we could exclude the plugin to copy for the file or folder ?

Please let me know

**Comment by eirslett** [comment]
I don't think this plugin is doing it, probably you're running some npm build tool that copies the files.

**Comment by raghavendra1221** [comment]
I think the working directory and installation Directory here are copying the files. Here if there is a way I can specify excludes, that will solve my problem. 

 <artifactId>frontend-maven-plugin</artifactId>
                <version>1.6</version>
                <configuration>
                    **<workingDirectory>**../ReactUI/src/main/resources/web</workingDirectory>
                    **<installDirectory**>target</installDirectory>
                </configuration>

**Comment by eirslett** [comment]
I suggest 1) use the latest version of the plugin, 2) remove all custom configuration of installDirectory, downloadDirectory, etc etc, and just use the plugin defaults. Does that work?
